tests/utils.go

```diff
@@ -4330,7 +4330,10 @@ func resetToDefaultConfig() {
 	}
 
 	UpdateKubeVirtConfigValueAndWait(KubeVirtDefaultConfig)
-	UpdateCDIConfigMap(CDIInsecureRegistryConfig)
+
+	if HasCDI() && CDIInsecureRegistryConfig != nil {
```

Comment:
a sufficient condition if preferred would be just to check
CDIInsecureRegistryConfig != nil

Reply:
I see that it is a part of the test setup - [here](https://github.com/kubevirt/kubevirt/blob/8a27f6e9a8747e474cb636e07b868579e5c58ac4/tests/utils.go#L754) and [here](https://github.com/kubevirt/kubevirt/blob/8a27f6e9a8747e474cb636e07b868579e5c58ac4/tests/utils.go#L779). Can you elaborate on why CDIInsecureRegistryConfig would be nil (maybe in the commit message)?

Reply:
its in the commit message, please open the link there and see that the Get is allowed to be NotFound

Reply:
Thanks!
/lgtm

Reply:
> a sufficient condition if preferred would be just to check
> `CDIInsecureRegistryConfig != nil`

Here is what I messed up. I am already checking on `nil`. but the `Get` call of the config map does not return `nil` even if it returns an error. Can we instead just ensure that `CDIInsecureRegistryConfig` stays really nil if we get a `errors.IsNotFound(err)`

Reply:
Sure on it, thanks

Reply:
Done